### ISSUE TITLE
[UX] Better intruction what you have to do to make it work, no error message

### ISSUE THREAD

**Issue by rugk** [AtomLinter/linter-eslint#378]
AFAIK node.js, npm and such things are bundled with Atom so you may not be required to set it up, but there seem to be settings for adding paths, which is a bit confusing.

And [you say](https://github.com/AtomLinter/linter-eslint#installation) eslint is bundled with this package, so I thought it should work out-of-the-box. But it does not.

In the settings I could find a setting which prevents execution when no ESLint config file is found and I disabled it and thought it would use some defaults, but there were still no linter errors.

Finally I also do not like that it does not show any error messages as it obviously does not work "out-of-the-box".

**Comment by SpainTrain** [comment]
Thank you for the feedback.

So you do not have an `.eslintrc` file?  eslint > 1.0 doesn't have any rules by default - you have to use `extends` or add them individually.  The option you mention was added before this linter used 1.0, so it made more sense at the time.

@AtomLinter/linter-eslint Thoughts on removing this option and instead issuing a warning or something when no `.eslintrc` file is found?

**Comment by SpainTrain** [comment]
cc @steelbrain @Arcanemagus @iam4x

**Comment by IanVS** [comment]
Yeah, come to think of it, it's probably worthless at this point. A better option would be to disable linting if a config file is not found within the folders open in the editor. Even then, I'm not sure how useful it would be.

**Comment by rugk** [comment]
[**@SpainTrain**](https://github.com/SpainTrain) commented on [20. Dez. 2015, 22:46 MEZ](https://github.com/AtomLinter/linter-eslint/issues/378#issuecomment-166156899 "2015-12-20T21:46:22Z - Replied by Github Comment Enhancer"):
> So you do not have an `.eslintrc` file?  eslint &gt; 1.0 doesn't have any rules by default - you have to use `extends` or add them individually.  The option you mention was added before this linter used 1.0, so it made more sense at the time.

No I do not have such a file. And as it seems I need one. :smile: 
So are there somewhere some suggestions for good "defaults"/most-often used/best rules or similar things? (You may also add this to the Readme - for noobies like me :wink:)
Or any guide/tool to create them? [EsLint](http://eslint.org/docs/user-guide/getting-started#usage) seems to have an "init" command, but how can I use this with the built-in version in Linter-EsLint? (Possibly add a command in the UI for this too?)

**Comment by rugk** [comment]
Ah okay found a good default one... :smiley: 

```json
{
    "extends": "eslint:recommended"
}
```

**Comment by IanVS** [comment]
I have a blog post which may be helpful for getting started with ESLint. http://devnull.guru/get-started-with-eslint/

**Comment by SpainTrain** [comment]
In general I recommend having the Cli tools for linters you are using installed.  This is always helpful for troubleshooting and sanity checks.

I agree about having helpful information in the README for folks new to eslint.  It could be as simple as a bulleted list of "getting started with eslint" links, including @IanVS's blog post.  Feel free to open a PR for this with what you found helpful @rugk, since you just went through it :smile_cat:

**Comment by Arcanemagus** [comment]
I agree that option should probably be removed, if a configuration can't be found we should just put a Warning "message" on the first line about needing a configuration.

**Comment by Arcanemagus** [comment]
@steelbrain brought up a good point, this would get very spammy on files/projects that aren't supposed to use `eslint`, how about changing the option to "Warn when no configuration exists"?

**Comment by steelbrain** [comment]
Fixed by #381
Thanks @rugk

**Comment by rugk** [comment]
Well... the thing with the option is now fixed...
But I think we can track this in another issue: https://github.com/AtomLinter/linter-eslint/issues/383
